path_planning:
  resolution: 0.15
  num_opt_points: 7
  step_size: 0.5
  rrt_factor: 1.15
  max_solve_t: 6
  uav_radius: 0.5
  clear_distance: 1.2
  max_check_dist: 1.75

# Lee position controller parameters
controllers:
  position_gain: [6, 6, 6]
  velocity_gain: [4.7, 4.7, 4.7]
  attitude_gain: [2, 3, 0.15]
  angular_rate_gain: [0.4, 0.52, 0.18]

global:
  dt: 0.5
  initial_alt: 1.0

mission:
  use_wp: true
  limit: [1, 2, 0, 0]
  cones:
    start: [0, 0, 1]
    middle: [20, -15, 1]
    stop: [0, -30, 1]
  forest:
    start: [0, 0, 0.4]
    middle: [15, -15, 0.4]
    stop: [20, -30, 0.5]
  big_forest:
    start: [0, 0, 0.5]
    middle: [35, -35, 0.5]
    stop: [70, -70, 0.5]
